### PR TITLE
RSDK-2239 Remove Pre-Map Representation API Endpoints from SLAM

### PR DESCRIPTION
**Pull request by jeremyrhyde**
This PR removes the pre-map representation API endpoints from the SLAM proto. These include `GetMap`, `Position`, and `GetInternalState` as well as associated messages.

**Note: This PR can only be merged after these endpoints are removed in rdk and carto/orb:**
- https://viam.atlassian.net/browse/RSDK-2242
- https://viam.atlassian.net/browse/RSDK-2215
- https://viam.atlassian.net/browse/RSDK-2214

Note #2: This PR has breaking changes and as such @tessavitabile will need to approve once we are ready.

JIRA Ticket: [RSDK-2239](https://viam.atlassian.net/browse/RSDK-2239)

[RSDK-2239]: https://viam.atlassian.net/browse/RSDK-2239?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: proto/viam/service/slam/v1/slam.proto [hunk 1]
```diff
@@ -137,44 +60,17 @@ message GetPositionNewResponse {
...
 }
 
 message GetPointCloudMapStreamResponse {
   // One chunk of the PointCloud.
-  // For a given GetPointCloudMapStream request,
-  // concatinating all
-  // GetPointCloudMapStreamResponse.point_cloud_pcd_chunk
-  // values in the order received result in
-  // the complete pointcloud in standard PCD format.
+  // For a given GetPointCloudMapStream request, concatinating all
```
Comment (tessavitabile) [inline proto/viam/service/slam/v1/slam.proto:70]: While you're here, might as well fix the spelling of "concatenating".

**Review by tessavitabile — APPROVED** [review]

File: proto/viam/service/slam/v1/slam.proto [hunk 2]
```diff
@@ -11,22 +11,6 @@ option java_package = "com.viam.service.slam.v1";
 
 // A SlamService declares the gRPC contract for a slam service
 service SLAMService {
-  // GetPosition returns the current estimated position of the robot with
```
Comment (EshaMaharishi) [inline proto/viam/service/slam/v1/slam.proto:14]: [note to self] the change to slam.proto is the only one we actually typed; the changes to the rest of the files in this PR were auto-generated based on that

File: app/v1/robot.pb.go [hunk 3]
```diff
@@ -986,6 +986,8 @@ func (x *AuthHandlerWebOauthConfig) GetJwks() *JWKSFile {
 	return nil
 }
 
+// ExternalAuthConfig describes how a viam managed robot can accept
+// credentials signed by the cloud app.
```
Comment (EshaMaharishi) [inline app/v1/robot.pb.go:990]: was this change intentional?

File: component/audioinput/v1/audioinput.pb.gw.go [hunk 4]
```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
+	v1_0 "go.viam.com/api/common/v1"
```
Comment (EshaMaharishi) [inline component/audioinput/v1/audioinput.pb.gw.go:18]: why did switch from calling it v1 to v1_0 here, but in other files we went from calling it v1_0 to v1?

**Review by EshaMaharishi — APPROVED** [review]
LGTM with non-blocking questions

**Review by nicksanford — CHANGES_REQUESTED** [review]

**Review by JohnN193 — APPROVED** [review]

File: app/v1/robot.pb.go (hunk 3 above)
Comment (jeremyrhyde) [reply app/v1/robot.pb.go]: Removed from PR

File: component/audioinput/v1/audioinput.pb.gw.go (hunk 4 above)
Comment (jeremyrhyde) [reply component/audioinput/v1/audioinput.pb.gw.go]: Removed from PR

**Comment by nicksanford** [comment]
moved to: https://github.com/viamrobotics/api/pull/224
